config/management/genesis/src/command.rs

```diff
@@ -0,0 +1,525 @@
+            _ => Err(self.unexpected_command(CommandName::CreateWaypoint)),
+        }
+    }
+
+    pub fn genesis(self) -> Result<Transaction, Error> {
+        match self {
+            Command::Genesis(genesis) => genesis.execute(),
+            _ => Err(self.unexpected_command(CommandName::Genesis)),
+        }
+    }
+
+    pub fn insert_waypoint(self) -> Result<Waypoint, Error> {
+        match self {
+            Command::InsertWaypoint(insert_waypoint) => insert_waypoint.execute(),
+            _ => Err(self.unexpected_command(CommandName::InsertWaypoint)),
+        }
+    }
+
+    pub fn operator_key(self) -> Result<Ed25519PublicKey, Error> {
+        match self {
+            Command::OperatorKey(operator_key) => operator_key.execute(),
+            _ => Err(self.unexpected_command(CommandName::OperatorKey)),
+        }
+    }
+
+    pub fn owner_key(self) -> Result<Ed25519PublicKey, Error> {
+        match self {
+            Command::OwnerKey(owner_key) => owner_key.execute(),
+            _ => Err(self.unexpected_command(CommandName::OwnerKey)),
+        }
+    }
+
+    pub fn read_account_state(self) -> Result<libra_types::account_state::AccountState, Error> {
+        match self {
+            Command::ReadAccountState(read_account_state) => read_account_state.execute(),
+            _ => Err(self.unexpected_command(CommandName::ReadAccountState)),
+        }
+    }
+
+    pub fn set_layout(self) -> Result<libra_management::layout::Layout, Error> {
+        match self {
+            Command::SetLayout(set_layout) => set_layout.execute(),
+            _ => Err(self.unexpected_command(CommandName::SetLayout)),
+        }
+    }
+
+    pub fn set_operator(self) -> Result<String, Error> {
+        match self {
+            Command::SetOperator(set_operator) => set_operator.execute(),
+            _ => Err(self.unexpected_command(CommandName::SetOperator)),
+        }
+    }
+
+    pub fn submit_transaction(self) -> Result<(), Error> {
+        match self {
+            Command::SubmitTransaction(submit_transaction) => submit_transaction.execute(),
+            _ => Err(self.unexpected_command(CommandName::SubmitTransaction)),
+        }
+    }
+
+    pub fn validator_config(self) -> Result<Transaction, Error> {
+        match self {
+            Command::ValidatorConfig(config) => config.execute(),
+            _ => Err(self.unexpected_command(CommandName::ValidatorConfig)),
+        }
+    }
+
+    pub fn verify(self) -> Result<String, Error> {
+        match self {
+            Command::Verify(verify) => verify.execute(),
+            _ => Err(self.unexpected_command(CommandName::Verify)),
+        }
+    }
+
+    fn unexpected_command(self, expected: CommandName) -> Error {
+        Error::UnexpectedCommand(expected.to_string(), CommandName::from(&self).to_string())
+    }
+}
+
+/// These tests depends on running Vault, which can be done by using the provided docker run script
```

Comment:
nit: do you mind removing these comments? I think this is out-of-date as the tests now use onDiskStorage. 'cc @davidiw 👀

Reply:
I won't remove them right now but we can in the future.